File: 10 - Python/Exercícios/04 - Vetores/Exercício 04.py
#Exercício - Números Pares

pares = 0

N = int(input("Quantos números você vai digitar? "))

vet = [0] * N

for i in range(N):

    vet[i] = int(input("Digite um número: "))

print("")
print("Números pares:")

for i in range(N):
    
    if vet[i] % 2 == 0:
        print(vet[i], "|", end=" ")
        pares = pares + 1

print("")
print("")
print("Quantidade de pares =", pares)

File: 07 - C++/Exercícios/04 - Vetores/Exercício 04.cpp
//Exercício - Números Pares

#include <iostream>

using namespace std;

int main(){

    int i, N, pares;

    pares = 0;

    cout << "Quantos numeros voce vai digitar? ";
    cin >> N;

    int vet[N];

    for(i = 0; i < N; i++){

        cout << "Digite um numero: ";
        cin >> vet[i];
    }

    cout << endl << "Numeros Pares:" << endl;

    for(i = 0; i < N; i++){

        if(vet[i] % 2 == 0){
            cout << vet[i] << " ";
            pares++;
        }
    }

    cout << endl << endl << "Quantidade de pares = " << pares;

    return 0;

}